devTools/tsconfigs/tsconfig.base.json

```diff
@@ -12,12 +12,8 @@
         "declaration": true,
         "declarationMap": true,
 
-        // We start with https://github.com/tsconfig/bases/blob/master/bases/node14.json then drop:
-        //   "lib": ["es2020"],
-        //   "target": "es2020",
-
-        "lib": ["dom", "dom.iterable", "es2020", "es2021", "es2022", "es2023"],
-        "target": "es2019",
+        "lib": ["dom", "dom.iterable", "ESNext"],
+        "target": "ES2022",
```

Comment:
This [should be](https://github.com/tsconfig/bases/blob/main/bases/node22.json) safe for Node 22.